repose-aggregator/components/filters/keystone-v2-filter/src/main/scala/org/openrepose/filters/keystonev2/KeystoneRequestHandler.scala

```diff
@@ -186,7 +197,8 @@ class KeystoneRequestHandler(identityServiceUri: String, akkaServiceClient: Akka
       s"$identityServiceUri${GROUPS_ENDPOINT(forToken)}",
       (Map(CommonHttpHeader.AUTH_TOKEN.toString -> authenticatingToken,
         CommonHttpHeader.ACCEPT.toString -> MediaType.APPLICATION_JSON)
-        ++ traceId.map(CommonHttpHeader.TRACE_GUID.toString -> _)).asJava))
+        ++ traceId.map(CommonHttpHeader.TRACE_GUID.toString -> _)).asJava,
+      checkCache))
```

Comment:
Not really, it's still all over this file. Does it really have to be passed through a million methods?